python/users.py

```diff
@@ -63,6 +63,8 @@ class User(sdk.python.entities.Entity):
     enable_store_notifications = Property(bool)
     is_super_user = Property(bool)
     is_admin_of_subscribed_company = Property(bool)
+    registered_as_guest = Property(bool)
+    registered_under_domain = Property(int)
```

Comment:
this one should be a domain rather than number

Reply:
I didn't want to add another relationship to the user. I only really needed the id ; making it a relationship is superfluous.